embeds/ge2017-nugget/_source/main.js

```diff
@@ -1,88 +1,127 @@
+        var thrasher = document.querySelector('.ge2017-supporter-nugget');
 
-    console.log('target:', targetContainer);
+        // the thrasher's section tag
+        var thrasherContainer = closest(thrasher, 'section');
 
+        // section where the treat should go
+        var targetContainer = findTargetContainer(thrasherContainer);
 
-    // get thrasher innerHTML
-    var thrasherWrap = document.createElement('div');
-    thrasherWrap.appendChild(thrasher.cloneNode(true));
-    var thrasherHTML = thrasherWrap.innerHTML;
+        // get thrasher innerHTML
+        var thrasherWrap = document.createElement('div');
+        thrasherWrap.appendChild(thrasher.cloneNode(true));
+        var thrasherHTML = thrasherWrap.innerHTML;
 
-    // clone thrasher into newTreat
-    var newTreat = document.createElement('li');
-    newTreat.classList.add('treats__list-item');
-    newTreat.innerHTML = thrasherHTML;
+        // clone thrasher into newTreat
+        var newTreat = document.createElement('li');
+        newTreat.classList.add('treats__list-item');
+        newTreat.innerHTML = thrasherHTML;
 
-    // find target container's treats
-    // or make it if it doesnt' exist
-    var containerTreats = targetContainer.querySelector('.treats__container');
-    if (!containerTreats) {
-        var newTreatList = document.createElement('ul');
-        newTreatList.classList.add('treats__container');
-        targetContainer.querySelector('.fc-container__inner').appendChild(newTreatList);
-        containerTreats = targetContainer.querySelector('.treats__container');
-    }
+        // find target container's treats
+        // or make it if it doesnt' exist
+        var containerTreats = targetContainer.querySelector('.treats__container');
+        if (!containerTreats) {
+            var newTreatList = document.createElement('ul');
+            newTreatList.classList.add('treats__container');
+            targetContainer.querySelector('.fc-container__inner').appendChild(newTreatList);
+            containerTreats = targetContainer.querySelector('.treats__container');
+        }
 
-    // inject on top if there are treats, at bottom otherwise
-    var firstTreat = containerTreats.querySelector('li:first-child');
-    if (firstTreat) {
-        containerTreats.insertBefore(newTreat, firstTreat);
-        firstTreat.classList.add('treat-separator');
-    } else {
-        containerTreats.appendChild(newTreat);
-    }
+        // inject on top if there are treats, at bottom otherwise
+        var firstTreat = containerTreats.querySelector('li:first-child');
+        if (firstTreat) {
+            containerTreats.insertBefore(newTreat, firstTreat);
+            firstTreat.classList.add('treat-separator');
+        } else {
+            containerTreats.appendChild(newTreat);
+        }
 
 
-    window.guardian.ophan.record({
-        component: 'thrasher : manifestos compared',
-        value: 1
-    });
-}, 20);
+        window.guardian.ophan.record({
```

Comment:
`window.guardian.ophan` doesn't necessarily exist. Could you please wrap it into:

```js
if (window.guardian.ophan && window.guardian.ophan.record) {
 /* ... */
}
```